pkg/app/client.go

```diff
@@ -0,0 +1,12 @@
+package app
+
+import "os/exec"
+
+func GetCOEVersion() ([]byte, error) {
+	return runCommand("oc", "version")
+}
+
+func runCommand(cmd string, args ...string) ([]byte, error) {
+	output, err := exec.Command(cmd, args...).CombinedOutput()
+	return output, err
+}
```

Comment:
We actually already have this segregated in `pkg/apb/hack.go`. Could make the function public and access it via `apb.RunCommand`, but that's not a semantically appropriate place for it.

I'd say we could put it in the `app` pkg, but we'll run into circular dependency problems with `app` importing `apb`, and `apb` importing `app`. It's a problem I've seen in a number of cases.

@jmrodri +1 on the utils pkg, although is it bad form to continue to add to our already sizable pkg count?